mono/mini/interp/interp.c

```diff
@@ -584,7 +584,9 @@ alloc_method_table (MonoVTable *vtable, int offset)
 	return table;
 }
 
-static InterpMethod*
+static
+MONO_NEVER_INLINE // Inlining causes additional stack use in caller.
```

Comment:
I'd really love it if we had these on the same line.

Reply:
And where is the comment? Above, nowhere (an ok answer), further to the right?
I like the function name at start so I can search for defs and decls, but not calls, with regexp ^foo.
If only we were consistent. :(
`static` and return type, either way.

Reply:
I admit disappointment in using outlining here. :(